client/signup/steps/domains/index.jsx

```diff
@@ -656,6 +686,9 @@ const submitDomainStepSelection = ( suggestion, section ) => {
 	if ( suggestion.isBestAlternative ) {
 		tracksObjects.label = 'best-alternative';
 	}
+	if ( abtest( 'prefilDomainStepValue' ) === 'test' && isBoolean( usedSuggestedDomain ) ) {
+		tracksObjects.usedSuggestedDomain = usedSuggestedDomain;
```

Comment:
I'm pretty sure tracks property names have to be `snake_case` otherwise they'll be rejected

Reply:
For reference: https://github.com/Automattic/wp-calypso/blob/master/client/lib/analytics/docs/tracks.md#naming-conventions